exercises/clock/clock_test.rb

```diff
@@ -3,146 +3,300 @@
   end
 
-  def test_negative_hours_and_minutes_both_roll_over_continuously
+  def test_negative_hour_and_minutes_both_roll_over
     skip
-    clock = Clock.at(0) - hours(121) - minutes(5810)
-    assert_equal '22:10', clock.to_s
+    assert_equal '20:20', Clock.at(-25, -160).to_s
+  end
+
+  def test_negative_hour_and_minutes_both_roll_over_continuously
+    skip
+    assert_equal '22:10', Clock.at(-121, -5810).to_s
+  end
+
+  def test_add_minutes
+    skip
+    assert_equal '10:03', (Clock.at(10, 0) + 3).to_s
   end
 
   def test_add_no_minutes
     skip
-    clock = Clock.at(0) + minutes(0)
-    assert_equal '00:00', clock.to_s
+    assert_equal '06:41', (Clock.at(6, 41) + 0).to_s
+  end
+
+  def test_add_to_next_hour
+    skip
+    assert_equal '01:25', (Clock.at(0, 45) + 40).to_s
+  end
+
+  def test_add_more_than_one_hour
+    skip
+    assert_equal '11:01', (Clock.at(10, 0) + 61).to_s
   end
 
   def test_add_more_than_two_hours_with_carry
     skip
-    clock = Clock.at(0, 45) + minutes(160)
-    assert_equal '03:25', clock.to_s
+    assert_equal '03:25', (Clock.at(0, 45) + 160).to_s
+  end
+
+  def test_add_across_midnight
+    skip
+    assert_equal '00:01', (Clock.at(23, 59) + 2).to_s
+  end
+
+  def test_add_more_than_one_day__1500_min_is_equal_to_25_hrs
+    skip
+    assert_equal '06:32', (Clock.at(5, 32) + 1500).to_s
   end
 
   def test_add_more_than_two_days
     skip
-    clock = Clock.at(1, 1) + days(2) + hours(10) + minutes(20)
-    assert_equal '11:21', clock.to_s
+    assert_equal '11:21', (Clock.at(1, 1) + 3500).to_s
+  end
+
+  def test_subtract_minutes
+    skip
+    assert_equal '10:00', (Clock.at(10, 3) + -3).to_s
+  end
+
+  def test_subtract_to_previous_hour
+    skip
+    assert_equal '09:33', (Clock.at(10, 3) + -30).to_s
+  end
+
+  def test_subtract_more_than_an_hour
+    skip
+    assert_equal '08:53', (Clock.at(10, 3) + -70).to_s
+  end
+
+  def test_subtract_across_midnight
+    skip
+    assert_equal '23:59', (Clock.at(0, 3) + -4).to_s
+  end
+
+  def test_subtract_more_than_two_hours
+    skip
+    assert_equal '21:20', (Clock.at(0, 0) + -160).to_s
+  end
+
+  def test_subtract_more_than_two_hours_with_borrow
+    skip
+    assert_equal '03:35', (Clock.at(6, 15) + -160).to_s
   end
 
-  def test_subtrackt_more_than_two_hours_with_borrow
+  def test_subtract_more_than_one_day__1500_min_is_equal_to_25_hrs
     skip
-    clock = Clock.at(6, 15) - hours(2) - minutes(40)
-    assert_equal '03:35', clock.to_s
+    assert_equal '04:32', (Clock.at(5, 32) + -1500).to_s
   end
 
   def test_subtract_more_than_two_days
     skip
-    clock = Clock.at(2, 20) - days(2) - hours(2)
-    assert_equal '00:20', clock.to_s
+    assert_equal '00:20', (Clock.at(2, 20) + -3000).to_s
   end
 
-  def test_equivalent_clocks
+  def test_clocks_with_same_time
     skip
     clock1 = Clock.at(15, 37)
     clock2 = Clock.at(15, 37)
-    assert_equal clock1, clock2
+    assert clock1.to_s == clock2.to_s
```

Comment:
Should we be able to compare these directly, instead of their string comparisons?